ferrocene/doc/internal-procedures/src/known-problems.rst

```diff
@@ -71,17 +79,10 @@ heading `Workaround`
 The `Workaround` section describes how to avoid running into the issue.
 This usually consists of rules that forbid certain code patterns.
 
-heading `Detection`
```

Comment:
Why are we removing this? Our KPs still have this section

Reply:
Sorry! I don't know why I thought those two sections had been merged together....